Source/Data/SpectrumAnalysis/KTSeqLine.cc

```diff
@@ -0,0 +1,273 @@
+/*
+ * KTScoredSpectrum.cc
+ *
+ *  Created on: Mar 10, 2016
+ *      Author: Christine
+ */
+
+#include <KTSeqLine.hh>
+
+#include "KTLogger.hh"
+
+#include <iostream>
+#include <cmath>
+
+
+namespace Katydid
+{
+	KTLOGGER(seqlog, "KTSeqLine");
+
+
+
+    LineRef::LineRef(const double& InitialSlope):
+        fStartTimeInRunC(0.0),
+        fStartTimeInAcq(0.0),
+        fEndTimeInRunC(0.0),
+        fEndTimeInAcq(0.0),
+        fStartFrequency(0.0),
+        fEndFrequency(0.0),
+        fInitialSlope(InitialSlope),
+        fSlope(0.0),
+        fNPoints(0),
+        fComponent(0),
+        fAmplitudeSum(0.0),
+        fAcquisitionID(0),
+        fProcTrackMinPoints(0),
+        fProcTrackAssError(0.),
+        fIntercept(0.),
+        fSlopeSigma(0.),
+        fInterceptSigma(0.),
+        fStartFrequencySigma(0.),
+        fEndFrequencySigma(0.)
+        {}
+
+    LineRef::~LineRef()
+    {}
+
+    void LineRef::InsertPoint(const Point& Point)
```

Comment:
1. Variables start with lower-case letters; to fix: `Point`

1. Instead of creating `newPoint` and then pushing it onto `fLinePoints`, which first constructs the `LinePoint` object then copies it into the vector, it's more efficient to use `std::vector::emplace_back`:
    ```C++
    fLinePoints.emplace_back(point.fBinINSlice, point.fPointFreq, . . . , point.fComponent);
    ```